http/main.py

```diff
@@ -163,6 +185,33 @@ async def data_from_multipart(request):
             meta_content["contentMetadata"]["private"][
                 "b64AuthenticatedMetadata"
             ] = base64.standard_b64encode(multipart_data["meta_raw"].encode()).decode()
+
+            # geolocation
+            geolocation = meta_content["contentMetadata"]["private"]["geolocation"]
+            meta = multipart_data["meta"]
+            geolocation["latitude"] = get_value_from_meta(
+                meta, "Current GPS Latitude"
+            ) or get_value_from_meta(meta, "Last Known GPS Latitude")
+            geolocation["longitude"] = get_value_from_meta(
+                meta, "Current GPS Longitude"
+            ) or get_value_from_meta(meta, "Last Known GPS Longitude")
+            geolocation["altitude"] = get_value_from_meta(
+                meta, "Current GPS Altitude"
+            ) or get_value_from_meta(meta, "Last Known GPS Altitude")
+            geolocation["timestamp"] = get_value_from_meta(
+                meta, "Current GPS Timestamp"
+            ) or get_value_from_meta(meta, "Last Known GPS Timestamp")
+            if geolocation["latitude"] is None or geolocation["longitude"] is None:
+                # No data, remove all the None (null) values
+                geolocation = {}
+            else:
+                # Add reverse-geocode keys
+                geolocation.update(
+                    geocode.reverse_geocode(
```

Comment:
1. Based on current implementation if `geocoder.osm([lat, lon], method="reverse").status_code != 200` it'd throw, but we are not catching here. The whole block will fail.
2. What if `reverse_geocode` returns None? Is that possible?

Reply:
1. Yes, the catching is done in the `create` function, so the whole request would fail and a 500 error would be returned the client. Should the request proceed without reverse-geolocation instead?
2. That is possible, but will raise a `TypeError`, so the same behaviour described above will occur.

Reply:
Discussed this in call. This is a user facing error and we want it to fail. @makew0rld will investigate if there are legitimate cases (e.g. the info about the lat/lon is actually not available) that leads to a thrown exception. We need to handle the legitimate cases, but ok to throw when e.g. service outage.

Reply:
Skip reverse-geocoding if lat/lon doesn't have a valid address. Merge this after Yurko reviews.